app/controllers/topics_controller.rb

```diff
@@ -0,0 +1,5 @@
+class TopicsController < ApplicationController
+	def index
```

Comment:
Unforgivable whitespace/indentation 🙄 
You can set default tab size (and to always use tabs instead of spaces) in your Sublime preferences